genie-proto/src/test/groovy/com/netflix/genie/proto/v4/adapters/JobSpecificationServiceAdapterSpec.groovy

```diff
@@ -416,6 +420,7 @@ class JobSpecificationServiceAdapterSpec extends Specification {
         def jobSpecification = com.netflix.genie.proto.JobSpecification
                 .newBuilder()
                 .setIsInteractive(interactive)
+                .setJobDirectoryLocation(jobDirectoryLocation.getPath())
```

Comment:
`getPath` or `getAbsolutePath` I'm never quite sure

Reply:
This should be `getAbsolutePath`, fixed.